### PR TITLE
ToDos in dc module

### PR DESCRIPTION
**Pull request by samsruti**
Since this file is not updated in edengci repo, so i am sending a pr to the main repo
Its for GCI task : http://www.google-melange.com/gci/task/view/google/gci2014/5346487999397888

### CODE REVIEW

**Comment by nursix** [comment]
Sorry, but this is nonsense - the "answer" field needs to be developed to store multi-type data.

I am currently working on this module, which is why these are "ToDos" - but they are to myself, not meant to be actioned. See status-tag: WIP

**Comment by samsruti** [comment]
Oops :( Sorry.. I wasnt completely sure about that .

File: modules/s3db/dc.py [hunk 1]
```diff
@@ -301,9 +303,8 @@ def model(self):
             msg_record_deleted = T("Data Collection deleted"),
             msg_list_empty = T("No Data Collections currently registered"))
 
-        # @todo: representation including template name, location and date
         represent = S3Represent(lookup=tablename,
-                                fields=["date"],
+                                fields=["template_id","location_id","date" ],
```
Comment (nursix) [inline modules/s3db/dc.py]: Ha ha - of course not the template_id and location_id. That doesn't tell the user anything.

**Comment by nursix** [comment]
I would request this to be left alone for now - I'll remove the status tag once I'm done with this.
Fine if you want to experiment with it, though (which is why it is shared in trunk)

**Comment by samsruti** [comment]
Sorry Sorry , I m trying some other todo

**Comment by nursix** [comment]
No need to apologize - it's not obvious.
But it's an incomplete implementation that needs development - no just "fixes".
I wasn't quite sure whether "answer" should become JSON, but that's the most likely solution - I left it a ToDo, so that I can find it more easily. However, there's a lot more to do here than the ToDos alone - this needs a widget, a representation and reporting capability - so your "fix" isn't really helpful, even if with good intention.
But if you want to experiment with XForms, then this is certainly something you can work with - input and feedback welcome.

**Comment by nursix** [comment]
Tip: best you start with ToDos where you are certain about the meaning.

Sometimes (or rather often) they are not meant to be executed "by the word", but actually require more in-depth thinking and development. If it was that easy, then it wouldn't have been a ToDo ;)

**Comment by samsruti** [comment]
Thanx @nursix  for your suggestions . I will surely implement that :+1:  :D

**Comment by nursix** [comment]
Implement what? :O

**Comment by samsruti** [comment]
Implement your suggestions :P

**Pull request closed without merging**